docs/book/src/cronjob-tutorial/testdata/emptymain.go

```diff
@@ -97,3 +98,32 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+/*
```

Comment:
I'd suggest move the new contents under line `mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{Scheme: scheme, MetricsBindAddress: metricsAddr})`.